web/webpack.config.js

```diff
@@ -49,6 +49,7 @@ module.exports = function (env) {
         "@src": path.resolve(__dirname, "src"),
         "@features": path.resolve(__dirname, "src/features"),
         "@stores": path.resolve(__dirname, "src/stores"),
+        "@types": path.resolve(__dirname, "src/types/index"),
```

Comment:
will need to update these aliases if we decide to break the types file up. for now i think it'll be fine to shove everything into index.ts